presto-docs/src/main/sphinx/release/release-0.227.rst

```diff
@@ -0,0 +1,60 @@
+=============
+Release 0.227
+=============
+
+General Changes
+_______________
+* Fix a bug where index joins would fail with the error ``driver should never block``.
+  Queries will now fail if the index is unable to load within the time specified by
+  the ``index-loader-timeout`` configuration property and the ``index_loader_timeout``
+  session property.
+* Fix a bug that could cause ``NullPointerException`` in spatial join with geometry
+  collections on the build side.
+* Improve performance for queries with ``FULL OUTER JOIN`` where join keys have the
+  ``COALESCE`` function applied.
+* Improve performance for ``UnnestOperator`` by using ``DictionaryBlock``.
+* Improve performance for ``PartitionedOutputOperator``. The new
+  ``OptimizedPartitionedOutputOperator`` can be enabled by the ``optimized_repartitioning``
+  session property or the ``experimental.optimized-repartitioning`` configuration property.
+* Add spatial join (broadcast and distributed) support for :func:`ST_Equals`,
+  :func:`ST_Overlaps`, :func:`ST_Crosses`, and :func:`ST_Touches`.
+* Add ``task_partitioned_writer_count`` session property to allow setting the number
+  of concurrent writers for partitioned (bucketed) writes.
+* Add ``IPPREFIX`` type and :func:`ip_prefix` function.
+* Add :func:`differential_entropy` functions to compute differential entropy.
+* Remove syntax support for ``SET PATH`` and ``CURRENT_PATH``. The path information was
+  never used by Presto.
+
+Hive Changes
+____________
+* Fix a bug that might lead to corruption when writing sorted table in the recoverable
+  grouped execution mode.
+* Fix ORC stripe skipping when using bloom filter.
+* Improve the CPU load on coordinator by reducing the cost of serializing ``HiveSplit``.
+* Improve GC pressure from Parquet reader by constraining the maximum column read size.
+  This can be configured by the ``parquet_max_read_block_size`` session property or the
+  ``hive.parquet.max-read-block-size`` configuration property.
+* Add support for sub-field pruning when reading Parquet files, so that only necessary
+  sub-fields are extracted from struct columns.
+* Add an option to switch file system on S3 to allow users to configure their own
```

Comment:
@yingsu00 This is not changed to what @caithagoras  suggested. The sentence is a bit long and hard to understand.

Reply:
The line got replicated. Thanks for catching it!

Reply:
The line got replicated. Thanks for catching it!